app/default/account_routes.py

```diff
@@ -101,6 +101,12 @@ def dashboard():
         for application in application_store_response
     ]
 
+    current_app.logger.info(
+        f"Applications :'{applications}'"
+    )
+
```

Comment:
Can we remove this logging I guess it was for debugging but doesn't really make sense to log all applications without context.

Reply:
Removed this